src/extension.ts

```diff
@@ -26,16 +26,18 @@ import * as debug from './debugger/calva-debug';
 import * as model from './cursor-doc/model';
 import * as outputWindow from './results-output/results-doc';
 import * as replHistory from './results-output/repl-history';
-import config from './config';
+import  { KEYBINDINGS_ENABLED_CONTEXT_KEY, KEYBINDINGS_ENABLED_CONFIG_KEY, 
```

Comment:
Again. I think we should rather pick it up from `config.` explicitly in the code below.